CHANGELOG.md

```diff
@@ -28,6 +29,7 @@ Documentation for hipRAND is available at
 
 ### Fixes
 
+* Fixed an issue in `rmake.py` where the list storing cmake options would contain individual characters instead of full string of option
```

Comment:
```suggestion
* Fixed an issue in `rmake.py` where the list storing cmake options would contain individual characters instead of a full string of options.
```